R/TerminatorRunTime.R

```diff
@@ -38,8 +37,34 @@ TerminatorRunTime = R6Class("TerminatorRunTime",
     #' @param archive ([Archive]).
     #' @return `logical(1)`.
     is_terminated = function(archive) {
+      if (is.null(archive$start_time)) {
+        return(FALSE)
```

Comment:
Can this happen? Looks "dangerous". So if no `start_time` was set, then we never terminate? Would it make more sense to throw an error?

Reply:
Oh this is a leftover from the old design. No this cannot happen anymore.